src/reporting/parameter.ts

```diff
@@ -153,6 +153,11 @@ export function createStackParametersQuickSight(scope: Construct, paramGroups?:
     default: 'Parameter Key Name',
   };
 
+  const redshiftServerlessIAMRoleParam = Parameters.createRedshiftServerlessDataRoleParameter(scope);
+  labels[redshiftServerlessIAMRoleParam.logicalId] = {
+    default: 'Redshift Serverless Data API Role ARN',
```

Comment:
remove `Serverless`, it's for both Redshift types

Reply:
If is is not  Serverless, we don't use this parameter in fact.

Reply:
We also need to query the data in Redshift if we are visualizing the exploration result.

Reply:
Exploration feature doesn't utilize data api connect to Redshift, in fact, it doesn't even need Redshift. However, I'll still rename it to avoid confusion.